src/main/java/haveric/recipeManager/recipes/CraftRecipeParser.java

```diff
@@ -88,7 +88,7 @@ public boolean parseRecipe(int directiveLine) throws Exception {
 
         recipe.setIngredients(ingredients); // done with ingredients, set 'em
 
-        if (recipe.hasFlag(FlagType.REMOVE)) {
+        if (recipe.hasFlag(FlagType.REMOVE) && !Version.has1_12Support()) {

```

Comment:
Let's take it out for now. Remove recipes _should_ work, but they'll need some more thorough testing after this is all pulled in.